src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ConfigurationElement.cs

```diff
@@ -1725,7 +1725,9 @@ private static object DeserializePropertyValue(ConfigurationProperty prop, XmlRe
                 // Validate the loaded and converted value
                 prop.Validate(propertyValue);
             }
+#pragma warning disable CA1859
```

Comment:
This is odd, why the suppression?

Reply:
it looked like a false positive to me because the analyzer suggested changing `ConfigurationException` to `ConfigurationErrorsException` but now that I looked closer `prop.ConvertFromString()` and `prop.Validate()`
indeed both throw `ConfigurationErrorsException`.

Will change it.